### PR TITLE
add-hints-and-commands

### PR DESCRIPTION
**Pull request by DorisW3**
I already added some hints and commands. Are they useful? Does anyone has some advices? (:

### CODE REVIEW

File: respectiv-session [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Hints and Commands
+
+
+##Useful hints
```
Comment (KilianBau) [inline respectiv-session]: Is there a space missing? :)

**Review by KilianBau — COMMENTED** [review]
Good one 👍

File: respectiv-session (hunk 1 above)
Comment (DorisW3) [reply respectiv-session]: yes, thank you! :-D

File: respectiv-session [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+~   |   Tide    |   option + N
```
Comment (jonas8900) [inline respectiv-session:6]: very Useful, nice syntax !

**Review by jonas8900 — COMMENTED** [review]
LGTM!

File: respectiv-session
Comment (JanaS87) [inline respectiv-session:1]: I like the layout and the hints.
Very useful  👍

File: respectiv-session (hunk 2 above)
Comment (DorisW3) [reply respectiv-session:6]: thanks!

File: respectiv-session (hunk 1 above)
Comment (DorisW3) [reply respectiv-session]: Have changed it.

**Pull request merged**